### PR TITLE
Ldanzinger/spatial operations

### PR DESCRIPTION
**Pull request by ldanzinger**
@michael-tims please review/merge

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/Geometry/SpatialOperations/SpatialOperations.cpp [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+#include "GeometryEngine.h"
+
+using namespace Esri::ArcGISRuntime;
+
+SpatialOperations::SpatialOperations(QQuickItem* parent /* = nullptr */):
+  QQuickItem(parent),
+  m_polygon1(Geometry()),
+  m_polygon2(Geometry()),
+  m_inputsOverlay (new GraphicsOverlay()),
+  m_outputsOverlay (new GraphicsOverlay())
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_CppSamples/Geometry/SpatialOperations/SpatialOperations.cpp]: The GraphicsOverlays will need parents, or need to be deleted in the destructor.

File: ArcGISRuntimeSDKQt_CppSamples/Geometry/SpatialOperations/SpatialOperations.pro [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    QMAKE_INFO_PLIST = $$PWD/Info.plist
+}
+
+android {
+    INCLUDEPATH += $$PWD
+    DEPENDPATH += $$PWD
+}
+
+macx {
+    QMAKE_MACOSX_DEPLOYMENT_TARGET = 10.11
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_CppSamples/Geometry/SpatialOperations/SpatialOperations.pro]: We shouldn't need `QMAKE_MACOSX_DEPLOYMENT_TARGET` anymore in samples.  It's being handled in the IDE integration pri files instead.

File: ArcGISRuntimeSDKQt_QMLSamples/Geometry/SpatialOperations/main.cpp [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+int main(int argc, char *argv[])
+{
+
+  QGuiApplication app(argc, argv);
+
+#ifdef Q_OS_WIN
+  // Force usage of OpenGL ES through ANGLE on Windows
+  QCoreApplication::setAttribute(Qt::AA_UseOpenGLES);
+#endif
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_QMLSamples/Geometry/SpatialOperations/main.cpp]: This ifdef should be removed for all 100.3 samples now.

**Comment by ldanzinger** [comment]
@michael-tims updated

**Pull request merged**